build errorの修正

## 概要
<!--
  この機能追加の目的・背景・解決したい課題を2〜3行で箇条書きにする。
  例:
  - ユーザーが〇〇できないため、△△が不便な状態
  - 〇〇機能を追加することで△△を解決する
-->

```
Run pnpm run build

> @ build /home/runner/work/keiba-db-gen2/keiba-db-gen2/source
> vite build

vite v8.0.3 building client environment for production...

transforming...✓ 2764 modules transformed.
✗ Build failed in 2.91s
error during build:
Build failed with 13 errors:

[UNLOADABLE_DEPENDENCY] Error: Could not load ../../../../../../resources/js/routes/password/confirm
   ╭─[ resources/js/pages/auth/confirm-password.tsx:8:23 ]
   │
 8 │ import { store } from "@/routes/password/confirm";
   │                       ─────────────┬─────────────  
   │                                    ╰─────────────── No such file or directory (os error 2)
───╯

[UNLOADABLE_DEPENDENCY] Error: Could not load ../../../../../../resources/js/routes/password
   ╭─[ resources/js/pages/auth/reset-password.tsx:9:24 ]
   │
 9 │ import { update } from "@/routes/password";
   │                        ─────────┬─────────  
   │                                 ╰─────────── No such file or directory (os error 2)
───╯

[UNLOADABLE_DEPENDENCY] Error: Could not load ../../../../../../resources/js/routes/verification
    ╭─[ resources/js/pages/settings/profile.tsx:12:22 ]
    │
 12 │ import { send } from "@/routes/verification";
    │                      ───────────┬───────────  
    │                                 ╰───────────── No such file or directory (os error 2)
────╯

[UNLOADABLE_DEPENDENCY] Error: Could not load ../../../../../../resources/js/routes/appearance
   ╭─[ resources/js/pages/settings/appearance.tsx:5:40 ]
   │
 5 │ import { edit as editAppearance } from "@/routes/appearance";
   │                                        ──────────┬──────────  
   │                                                  ╰──────────── No such file or directory (os error 2)
───╯

[UNLOADABLE_DEPENDENCY] Error: Could not load ../../../../../../resources/js/actions/App/Http/Controllers/Settings/ProfileController
   ╭─[ resources/js/pages/settings/profile.tsx:4:31 ]
   │
 4 │ import ProfileController from "@/actions/App/Http/Controllers/Settings/ProfileController";
   │                               ─────────────────────────────┬─────────────────────────────  
   │                                                            ╰─────────────────────────────── No such file or directory (os error 2)
───╯

...
    at aggregateBindingErrorsIntoJsError (file:///home/runner/work/keiba-db-gen2/keiba-db-gen2/source/node_modules/.pnpm/rolldown@1.0.0-rc.12_@emnapi+core@1.9.1_@emnapi+runtime@1.9.1/node_modules/rolldown/dist/shared/error-BLhcSyeg.mjs:48:18)
    at unwrapBindingResult (file:///home/runner/work/keiba-db-gen2/keiba-db-gen2/source/node_modules/.pnpm/rolldown@1.0.0-rc.12_@emnapi+core@1.9.1_@emnapi+runtime@1.9.1/node_modules/rolldown/dist/shared/error-BLhcSyeg.mjs:18:128)
    at #build (file:///home/runner/work/keiba-db-gen2/keiba-db-gen2/source/node_modules/.pnpm/rolldown@1.0.0-rc.12_@emnapi+core@1.9.1_@emnapi+runtime@1.9.1/node_modules/rolldown/dist/shared/rolldown-build-CPrIX9V6.mjs:3313:34)
    at async buildEnvironment (file:///home/runner/work/keiba-db-gen2/keiba-db-gen2/source/node_modules/.pnpm/vite@8.0.3_@emnapi+core@1.9.1_@emnapi+runtime@1.9.1_@types+node@22.19.15_esbuild@0.27.7_jiti@2.6.1/node_modules/vite/dist/node/chunks/node.js:32849:64)
    at async Object.build (file:///home/runner/work/keiba-db-gen2/keiba-db-gen2/source/node_modules/.pnpm/vite@8.0.3_@emnapi+core@1.9.1_@emnapi+runtime@1.9.1_@types+node@22.19.15_esbuild@0.27.7_jiti@2.6.1/node_modules/vite/dist/node/chunks/node.js:33271:19)
    at async Object.buildApp (file:///home/runner/work/keiba-db-gen2/keiba-db-gen2/source/node_modules/.pnpm/vite@8.0.3_@emnapi+core@1.9.1_@emnapi+runtime@1.9.1_@types+node@22.19.15_esbuild@0.27.7_jiti@2.6.1/node_modules/vite/dist/node/chunks/node.js:33268:153)
    at async CAC.<anonymous> (file:///home/runner/work/keiba-db-gen2/keiba-db-gen2/source/node_modules/.pnpm/vite@8.0.3_@emnapi+core@1.9.1_@emnapi+runtime@1.9.1_@types+node@22.19.15_esbuild@0.27.7_jiti@2.6.1/node_modules/vite/dist/node/cli.js:778:3) {
  errors: [Getter/Setter]
}
 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

## 機能要件
<!--
  実装すべき具体的な振る舞いをユーザー視点で箇条書きにする。
  「〜できる」「〜が表示される」など動詞を使って記述する。
  例:
  - ユーザーは〇〇画面で△△ボタンを押すと〜できる
  - システムは〇〇の場合に△△を返す
-->

## 制約
<!--
  実装時に守るべき非機能要件・技術的制約・スコープ外を箇条書きにする。
  例:
  - 既存の〇〇APIの仕様を変更しない
  - パフォーマンス要件: レスポンス〇〇ms以内
  - 今回のスコープ外: △△は別issueで対応
-->

--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -40,6 +40,11 @@ jobs:
       - name: Install dependencies
         run: pnpm install
 
+      - name: Generate frontend resources
+        run: |
+          php artisan ziggy:generate
+          php artisan route:cache
+
       - name: Build assets
         run: pnpm run build
 

